### ISSUE TITLE
Sprockets::FileNotFound

### ISSUE THREAD

**Issue by daniel-gato** [TheDartCode/typedjs-rails#3]
Hello,

I'm new to rails and I don't really know where to look for to solve it.

I added `//= require typed` in my application.js
I added `gem "typedjs-rails", "~> 1.0.4"` in my Gemfile

I've that in my Gemfile.lock

```
GEM
  remote: https://rubygems.org/
  specs:
    actioncable (5.0.0.1)
      actionpack (= 5.0.0.1)
      nio4r (~> 1.2)
      websocket-driver (~> 0.6.1)
    actionmailer (5.0.0.1)
      actionpack (= 5.0.0.1)
      actionview (= 5.0.0.1)
      activejob (= 5.0.0.1)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 2.0)
    actionpack (5.0.0.1)
      actionview (= 5.0.0.1)
      activesupport (= 5.0.0.1)
      rack (~> 2.0)
      rack-test (~> 0.6.3)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (5.0.0.1)
      activesupport (= 5.0.0.1)
      builder (~> 3.1)
      erubis (~> 2.7.0)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    activejob (5.0.0.1)
      activesupport (= 5.0.0.1)
      globalid (>= 0.3.6)
    activemodel (5.0.0.1)
      activesupport (= 5.0.0.1)
    activerecord (5.0.0.1)
      activemodel (= 5.0.0.1)
      activesupport (= 5.0.0.1)
      arel (~> 7.0)
    activesupport (5.0.0.1)
      concurrent-ruby (~> 1.0, >= 1.0.2)
      i18n (~> 0.7)
      minitest (~> 5.1)
      tzinfo (~> 1.1)
    arel (7.1.4)
    autoprefixer-rails (6.5.1.1)
      execjs
    bcrypt (3.1.11)
    bcrypt-ruby (3.1.5)
      bcrypt (>= 3.1.3)
    bootstrap-sass (3.3.7)
      autoprefixer-rails (>= 5.2.1)
      sass (>= 3.3.4)
    bootstrap-will_paginate (0.0.10)
      will_paginate
    builder (3.2.2)
    byebug (9.0.6)
    coffee-rails (4.2.1)
      coffee-script (>= 2.2.0)
      railties (>= 4.0.0, < 5.2.x)
    coffee-script (2.4.1)
      coffee-script-source
      execjs
    coffee-script-source (1.10.0)
    concurrent-ruby (1.0.2)
    erubis (2.7.0)
    execjs (2.7.0)
    ffi (1.9.14)
    globalid (0.3.7)
      activesupport (>= 4.1.0)
    i18n (0.7.0)
    jbuilder (2.6.0)
      activesupport (>= 3.0.0, < 5.1)
      multi_json (~> 1.2)
    jquery-rails (4.2.1)
      rails-dom-testing (>= 1, < 3)
      railties (>= 4.2.0)
      thor (>= 0.14, < 2.0)
    json (2.0.2)
    listen (3.0.8)
      rb-fsevent (~> 0.9, >= 0.9.4)
      rb-inotify (~> 0.9, >= 0.9.7)
    loofah (2.0.3)
      nokogiri (>= 1.5.9)
    mail (2.6.4)
      mime-types (>= 1.16, < 4)
    method_source (0.8.2)
    mime-types (3.1)
      mime-types-data (~> 3.2015)
    mime-types-data (3.2016.0521)
    mini_portile2 (2.1.0)
    minitest (5.9.1)
    multi_json (1.12.1)
    nio4r (1.2.1)
    nokogiri (1.6.8.1)
      mini_portile2 (~> 2.1.0)
    pg (0.19.0)
    puma (3.6.0)
    rack (2.0.1)
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (5.0.0.1)
      actioncable (= 5.0.0.1)
      actionmailer (= 5.0.0.1)
      actionpack (= 5.0.0.1)
      actionview (= 5.0.0.1)
      activejob (= 5.0.0.1)
      activemodel (= 5.0.0.1)
      activerecord (= 5.0.0.1)
      activesupport (= 5.0.0.1)
      bundler (>= 1.3.0, < 2.0)
      railties (= 5.0.0.1)
      sprockets-rails (>= 2.0.0)
    rails-dom-testing (2.0.1)
      activesupport (>= 4.2.0, < 6.0)
      nokogiri (~> 1.6.0)
    rails-html-sanitizer (1.0.3)
      loofah (~> 2.0)
    railties (5.0.0.1)
      actionpack (= 5.0.0.1)
      activesupport (= 5.0.0.1)
      method_source
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rake (11.3.0)
    rb-fsevent (0.9.8)
    rb-inotify (0.9.7)
      ffi (>= 0.5.0)
    sass (3.4.22)
    sass-rails (5.0.6)
      railties (>= 4.0.0, < 6)
      sass (~> 3.1)
      sprockets (>= 2.8, < 4.0)
      sprockets-rails (>= 2.0, < 4.0)
      tilt (>= 1.1, < 3)
    spring (2.0.0)
      activesupport (>= 4.2)
    spring-watcher-listen (2.0.1)
      listen (>= 2.7, < 4.0)
      spring (>= 1.2, < 3.0)
    sprockets (3.7.0)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.2.0)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)
    sqlite3 (1.3.12)
    thor (0.19.1)
    thread_safe (0.3.5)
    tilt (2.0.5)
    turbolinks (5.0.1)
      turbolinks-source (~> 5)
    turbolinks-source (5.0.0)
    typedjs-rails (1.0.4)
      railties (>= 3.1)
    tzinfo (1.2.2)
      thread_safe (~> 0.1)
    uglifier (3.0.3)
      execjs (>= 0.3.0, < 3)
    websocket-driver (0.6.4)
      websocket-extensions (>= 0.1.0)
    websocket-extensions (0.1.2)
    will_paginate (3.1.5)

PLATFORMS
  ruby

DEPENDENCIES
  bcrypt-ruby
  bootstrap-sass
  bootstrap-will_paginate
  byebug
  coffee-rails (~> 4.2)
  jbuilder
  jquery-rails (~> 4.2.1)
  json
  listen (~> 3.0.5)
  pg
  puma (~> 3.0)
  rails (~> 5.0.0, >= 5.0.0.1)
  sass-rails
  spring
  spring-watcher-listen (~> 2.0.0)
  sprockets
  sqlite3
  turbolinks (~> 5)
  typedjs-rails (~> 1.0.4)
  uglifier (>= 1.3.0)
  will_paginate

RUBY VERSION
   ruby 2.3.1p112

BUNDLED WITH
   1.13.6
```

and when I try to access my page I have :```
couldn't find file 'typed' with type 'application/javascript'
Checked in these paths: 
  /Users/Daniel/GitHub/miam-rails/app/assets/config
  /Users/Daniel/GitHub/miam-rails/app/assets/images
  /Users/Daniel/GitHub/miam-rails/app/assets/javascripts
  /Users/Daniel/GitHub/miam-rails/app/assets/stylesheets
  /Users/Daniel/GitHub/miam-rails/vendor/assets/javascripts
  /Users/Daniel/GitHub/miam-rails/vendor/assets/stylesheets
  /Users/Daniel/.rvm/gems/ruby-2.3.1/gems/jquery-rails-4.2.1/vendor/assets/javascripts
  /Users/Daniel/.rvm/gems/ruby-2.3.1/gems/coffee-rails-4.2.1/lib/assets/javascripts
  /Users/Daniel/.rvm/gems/ruby-2.3.1/gems/actioncable-5.0.0.1/lib/assets/compiled
  /Users/Daniel/.rvm/gems/ruby-2.3.1/gems/turbolinks-source-5.0.0/lib/assets/javascripts
  /Users/Daniel/.rvm/gems/ruby-2.3.1/gems/bootstrap-sass-3.3.7/assets/stylesheets
  /Users/Daniel/.rvm/gems/ruby-2.3.1/gems/bootstrap-sass-3.3.7/assets/javascripts
  /Users/Daniel/.rvm/gems/ruby-2.3.1/gems/bootstrap-sass-3.3.7/assets/fonts
  /Users/Daniel/.rvm/gems/ruby-2.3.1/gems/bootstrap-sass-3.3.7/assets/images
Extracted source (around line #17):
15
16
17
18

//= require bootstrap-sprockets
//= require turbolinks
//= require typed
//= require_tree .

Rails.root: /Users/Daniel/GitHub/miam-rails

Application Trace | Framework Trace | Full Trace
app/assets/javascripts/application.js:17
app/views/layouts/public.html.erb:8:in `_app_views_layouts_public_html_erb__86621966738206768_70357464275200'

```

Any ideas?
```

**Comment by TheDartCode** [comment]
Hey Daniel,
I tried to reproduce your problem but everything is working correctly.
Have
Can you perhaps send me your Gemfile?

**Comment by daniel-gato** [comment]
Thanks for your answer!

In fact, I think I know what it is. It happened with other gems too. When I add a gem and do a bundle install, I’ve to restart my the rails server.

I don’t know why, but it does it on rails v5 on my dev stations.

Thanks again!

Closes #3

**Comment by TheDartCode** [comment]
This is the way rails/sprockets works, it also happens in rails < v5 ;-)

Keep on!
